vision_evaluation/evaluators.py

```diff
@@ -1100,3 +1100,46 @@ def add_predictions(self, predictions, ground_truths):
 
     def get_report(self, **kwargs):
         return {self._get_id(): {group: eval.get_report(**kwargs) for group, eval in self._evaluators.items()}}
+
+
+class MeanLpErrorEvaluator(Evaluator):
+    """
+    Error evaluator for regression targets
+    Parameters:
```

Comment:
The parameters doc should be in the __init__ function

Reply:
Good catch, thanks!